src/cc/export/helpers.h

```diff
@@ -45,6 +45,9 @@ struct _name##_table_t _name
 #define BEGIN(next) \
   u64 _parse_cursor = 0; \
   goto next
+#define BEGIN_OFFSET(next, _base_offset) \
```

Comment:
begin and begin_offset probably need some renaming.
It's impossible to get the meaning what they suppose to do without looking at the sources.